Src/BootCamp.Chapter/Program.cs

```diff
@@ -9,6 +11,55 @@ class Program
     {
         static void Main(string[] args)
         {
+            string name = AskingAndConvertingSTRING();
+            string surname = AskingAndConvertingSTRING();
+            int age = AskingAndConvertingINT();
+            float weight = AskingAndConvertingFLOAT();
+            float height = AskingAndConvertingFLOAT();
+
+            Console.WriteLine(name + " " + surname + " is " + age + " years old, his weight is " + weight + " kg and his height is " + height + " cm.");
+
+            float bmi = CalculateBMI(weight, height);
+
+            Console.WriteLine("His BMI is " + bmi + ".");
+
+            // Up - first person, down - second.
+
+            string secondName = AskingAndConvertingSTRING();
+            string secondSurname = AskingAndConvertingSTRING();
+            int secondAge = AskingAndConvertingINT();
+            float secondWeight = AskingAndConvertingFLOAT();
+            float secondHeight = AskingAndConvertingFLOAT();
+
+            Console.WriteLine(secondName + " " + secondSurname + " is " + secondAge + " years old, his weight is " + secondWeight + " kg and his height is " + secondHeight + " cm.");
+
+            float secondBmi = CalculateBMI(secondWeight, secondHeight);
+
+            Console.WriteLine("His BMI is " + secondBmi + ".");
+
+            Console.ReadLine();
+
+        }
+        public static float CalculateBMI(float weight, float height)
```

Comment:
Don't have those methods in Program class. Program class is just for bootstrapping. These should be in their own class.

Reply:
First of all, is this a question of style? Or it is a requirement? Why would I take a class just for functions, like a bank?

Reply:
It's a style, it's not a requirement. As  I mentioned, having logic in Program class is out of place, it should just kickstart the application. No biggie for the first chapter though.